frontend/spec/features/checkout_confirm_insufficient_stock_spec.rb

```diff
@@ -8,7 +8,7 @@
   context "when the product from the order is not backorderable but has enough stock quantity" do
     let(:user) { create(:user) }
 
-    let(:order) { Spree::TestingSupport::OrderWalkthrough.up_to(:payment) }
+    let(:order) { Spree::TestingSupport::OrderWalkthrough.up_to(:payment, user) }
```

Comment:
Would read much better as `up_to(:payment, user: user)`

Reply:
I've extracted that commit into its own PR https://github.com/solidusio/solidus/pull/4292